### PR TITLE
Added default options to script initialization

### PR DESCRIPTION
**Pull request by pablovilas**
* Added config options
* Updated documentation
* Added suggestions of previous pr

### CODE REVIEW

File: lib/ServerDate.js [hunk 1]
```diff
@@ -43,7 +43,37 @@ var
   precision,
   offset,
   target = null,
-  synchronizing = false;
+  synchronizing = false,
+
+  // ServerDate default configuration
+  config = {
```
Comment (NodeGuy) [inline lib/ServerDate.js]: I don't want this extra variable and the following code to merge it.  Please keep the options as they were before.

**Comment by NodeGuy** [comment]
Thank you.  Next time please update the existing pull request instead of creating a new one.

File: lib/ServerDate.js (hunk 1 above)
Comment (pablovilas) [inline lib/ServerDate.js]: Do you want to remove the configuration options?. The only change will be the HEAD request instead of GET

Comment (NodeGuy) [inline lib/ServerDate.js]: No, I like the new `debug` option.  Keep that.

**Pull request closed without merging**